src/modules/storage/Storages.ts

```diff
@@ -0,0 +1,54 @@
+/*******************************************************************************
+
+    The superclass of storages.
+
+    This has a DB instance.
+
+    Copyright:
+        Copyright (c) 2020 BOS Platform Foundation Korea
+        All rights reserved.
+
+    License:
+        MIT License. See LICENSE for details.
+
+*******************************************************************************/
+
+import * as sqlite from 'sqlite3';
+
+export class Storages
+{
+    /**
+     *  The instance of sqlite3
+     */
+    protected db: sqlite.Database;
+
+    /**
+     * Constructor
+     * @param filename Valid values are filenames,
+     * ":memory:" for an anonymous in-memory database and
+     * an empty string for an anonymous disk-based database
+     * @param callback : If provided, this function will be called when
+     * the database was opened successfully or when an error occurred.
+     * The first argument is an error object.
+     */
+    constructor (callback: any = null)
+    {
+        let filename: string = ".cache";
```

Comment:
I still don't understand what you're trying to accomplish.

You will have multiple instances of `Storages` all writing to the same file?

Reply:
Yes, multiple stores are managed in a single file.
I'd like to join a number of tables and use them.

Reply:
Is it because the name storage is strange?

Reply:
Then why do you need multiple instances of `sqlite.Database`? If you will only use one file then you only need one database handle.

Reply:
I think this is what @AndrejMitrovic  wants.
I think this is good too.

```suggestion
    constructor (filename: string, callback?: any)
```

Reply:
Yes that is what I thought Henry tried to do at first. But according to him he wants a single file for all the tables..

Reply:
I'll leave design decisions to you two. It was just a comment.

Reply:
Maybe @AndrejMitrovic  talking about a single-ton design pattern. Is that right?

Reply:
Then it should get a DB Instance through `getDbInstancea`
In subsequent operations, you can manage instances as one after creating superclass in `Stoa.ts`

It is still applied only to the test class.